openedx/core/djangoapps/certificates/tests/test_api.py

```diff
@@ -1,187 +0,0 @@
-"""
-API tests for the openedx certificates app
-"""
-from contextlib import contextmanager
-from datetime import datetime
-
-import ddt
-import pytz
-from django.test import TestCase
-from unittest.mock import patch
-from edx_toggles.toggles import LegacyWaffleSwitch
-from edx_toggles.toggles.testutils import override_waffle_switch
-
-from common.djangoapps.course_modes.models import CourseMode
-from common.djangoapps.student.tests.factories import CourseEnrollmentFactory, UserFactory
-from openedx.core.djangoapps.certificates import api
-from openedx.core.djangoapps.certificates.config import waffle as certs_waffle
-from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
-from xmodule.data import CertificatesDisplayBehaviors
-from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
-
-
-BETA_TESTER_METHOD = 'openedx.core.djangoapps.certificates.api.access.is_beta_tester'
-CERTS_VIEWABLE_METHOD = 'openedx.core.djangoapps.certificates.api.certs_api.certificates_viewable_for_course'
-PASSED_OR_ALLOWLISTED_METHOD = 'openedx.core.djangoapps.certificates.api._has_passed_or_is_allowlisted'
-
-
-# TODO: Copied from lms.djangoapps.certificates.models,
-# to be resolved per https://openedx.atlassian.net/browse/EDUCATOR-1318
-class CertificateStatuses:
```

Comment:
Note: This was the one thing I didn't copy over because the real one was already in the new file.